src/Core/Package.vala

```diff
@@ -297,6 +297,23 @@ public class AppCenterCore.Package : Object {
         }
     }
 
+    private string? _author_id = null;
+    public string? author_id {
+        get {
+            if (_author_id != null) {
+                return _author_id;
+            }
+
+            _author = component.get_developer ().get_id ();
```

Comment:
```suggestion
            _author_id = component.get_developer ().get_id ();
```

Maybe a copy & paste error

Reply:
True. I'll change this a little bit just to include your other suggestion.